test/labeling/test_utils.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+
+import numpy as np
+import pandas as pd
+
+from snorkel.labeling.utils import filter_unlabeled
+
+
+class TestAnalysis(unittest.TestCase):
+    def test_filter_unlabeled(self) -> None:
+        X = pd.DataFrame([["x", 1], ["y", 2], ["z", 3]], columns=["A", "B"])
+        y = np.array([[0.4, 0.3, 0.3], [1.0, 0.0, 0.0], [0.2, 0.3, 0.5]])
```

Comment:
To defend against someone trying to do this with just `y`, can we switch to 4 classes and add a row that's [0.25, 0.25, 0.25, 0.25] but the corresponding label matrix row isn't all abstain?